cranelift/codegen/src/isa/arm32/mod.rs

```diff
@@ -140,6 +140,15 @@ impl TargetIsa for Isa {
     fn as_any(&self) -> &dyn Any {
         self as &dyn Any
     }
+
+    fn is_compatible_with(&self, other: &dyn TargetIsa) -> bool {
+        match other.as_any().downcast_ref::<Self>() {
+            Some(other) => {
+                self.triple == other.triple && self.isa_flags.is_compatible_with(&other.isa_flags)
+            }
```

Comment:
I ended up omitting the `self.shared_flags.is_compatible_with(&other.shared_flags)` here because here I'm saying "self.isa_flags is more specific than other.isa_flags" (e.g. the host machine has more ISA flags set than the filetest will) but that same logic doesn't apply to the shared flags (e.g. the default shared flags will almost always have less flags set than the filetest).

Reply:
And here's an example of the skipped tests on Windows: https://github.com/bytecodealliance/wasmtime/pull/1916/checks?check_run_id=804670568#step:12:652.